sdk/core/azure-core/tests/test_pipeline.py

```diff
@@ -264,6 +264,25 @@ def test_request_url_with_params(self):
 
         self.assertIn(request.url, ["a/b/c?g=h&t=y", "a/b/c?t=y&g=h"])
 
+    def test_request_text(self):
+        client = PipelineClientBase('http://example.org')
+        request = client.get(
+            "/",
+            content="foo"
+        )
+
+        # In absence of information, everything is JSON (double quote added)
+        assert request.data == json.dumps("foo")
+
+        request = client.get(
```

Comment:
Please change to POST. You are passing in a request body in a GET method. While it matters about 0% in this case, it still irks me :)

Reply:
Updated 😃